### DISCUSSION TITLE
HMS-800-2T  läuft nicht

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#970] [General]
Hallo Leute,

wollte jetzt bei meinem Nachbarn den Hoymiles in Betrieb nehmen.
Erstmal eine Platte an einem Eingang und den Ausgang normal auf die AußenSteckdose.
DC und AC Versorgung passt.

DC ran.... blinkt 5x grün und dann Rot jede Sekunde.
Hab auch mal 20 Minuten gewartet, ändert sich nix...

Musste man, bevor die laufen nicht das Netzprofil angeben?
Ich finde die Stelle aber nirgends in der Installer App 🤷‍♂️

**Comment by Farquarl** [comment]
Welche Spannung wird von dem Panel bereitgestellt? Wenn es ein 12v panel ist schließe ein zweites in Reihe.

**Reply by rs3hc** [reply]
Danke für den Tipp,  aber nein.
Es sind Longi 430W Full Black mit 39V Leerlaufspannung.
Das muss passen.

Er erkennt ja scheinbar das Netz nicht.
Oder er ist schlicht defekt

**Comment by tonibbb** [comment]
Aus dem Handbuch: "Rotes Blinken (1 s Intervall): Keine Stromerzeugung aufgrund eines ungültigen AC-Netzes". Spannung an der Steckdose passt?

**Reply by rs3hc** [reply]
Habe ich schon im Startthread geschrieben.
Ja AC passt (Bin Ekektriker)

Hatte gehofft ich habe irgendeine Einstellung übersehen.

Meiner läuft auch 1a 🤷‍♂️
